scopes/react/react/docs/base.tsx

```diff
@@ -75,33 +45,30 @@ export function Base({ docs = defaultDocs, componentId, compositions, ...rest }:
   const Content: any = isFunction(docs.default) ? docs.default : () => null;
 
   return (
-    <ThemeContext>
```

Comment:
loading/unloading themeContext and the icon font when gql finished made the browser work harder, and more likely to freeze. so, moved it to docs-app

Reply:
I'm considering to move the whole gql together with ComponentOverview to a separate component. Not sure if it's ideal though